codegen/html.js

```diff
@@ -36,6 +36,9 @@ const printRow = (e, elProps) =>
   elProps.length > 0
     ? `
   ( _data :: Object String
+  -- ARIA
+  , ${ariaProps.map(p => `${p} :: String`).join("\n  , ")}
```

Comment:
These are invalid record labels. You have to double-quote them:

```suggestion
  , ${ariaProps.map(p => `"${p}" :: String`).join("\n  , ")}
```

Reply:
Right, I haven’t yet had a chance to test this, but will before merging 👍